.github/workflows/ci.yml

```diff
@@ -134,6 +134,7 @@ jobs:
           benchmark_bulk_xt,
           compose_extrinsic_offline,
           custom_nonce,
+          dispatch_errors_tests,
```

Comment:
Could you move this down to the other test files? Probably helps to keep an overview